examples/shapes/shapes_math_sine_cosine.c

```diff
@@ -0,0 +1,180 @@
+#define RAYGUI_IMPLEMENTATION
+#include "raygui.h"
+
+#include <math.h>
+
+int main(void)
+{
+    // Window initialization
+    const int screenWidth = 800;
+    const int screenHeight = 450;
+
+    InitWindow(screenWidth, screenHeight, "raylib [shapes] example - math sine cosine");
+    const char* playText = "Play";
+    const char* pauseText = "Pause";
+
+    // Scene layout (mimic p5 example positioning)
+    const float circleX = 200.0f;
+    const float circleY = 150.0f;
+    const float circleRadius = 75.0f;
+
+    const float graphX = 50.0f;
+    const float graphY = 300.0f;
+    const float graphAmplitude = 50.0f;
+    const float graphPeriod = 300.0f; // width of the graph in pixels
+
+    // Animation / UI state
+    bool playing = true;     // play / pause toggle
+    int frameCountLocal = 0; // local frame counter (used when playing)
+
+    SetTargetFPS(60);
+
+    // Main loop
+    while (!WindowShouldClose())
+    {
+        // Handle GUI and input first
+        BeginDrawing();
+
+        ClearBackground(BLACK);
+
+        // Right control panel
+        float panelX = (float)GetScreenWidth() - 270.0f;
+        Rectangle panel = {panelX, 10, 260, (float)GetScreenHeight() - 20};
+        DrawRectangleRec(panel, Fade(LIGHTGRAY, 0.6f));
+        DrawRectangleLinesEx(panel, 1, GRAY);
+
+        int y = 20;
+        int px = (int)panelX + 10;
+        DrawText("Sine / Cosine Demo", px, y, 20, DARKGRAY);
+        y += 34;
+
+        // Play / Pause toggle
+        DrawText("Animation:", px, y, 14, DARKGRAY);
+        y += 18;
+        GuiToggle((Rectangle){(float)px, y, 220, 30}, playing ? pauseText : playText, &playing);
+        y += 40;
+
+        // Small description
+        DrawText("Animated demonstration of a point moving", px, y, 10, DARKGRAY);
+        y += 14;
+        DrawText("around the unit circle and the corresponding", px, y, 10, DARKGRAY);
+        y += 14;
+        DrawText("sine (red) and cosine (orange) graphs.", px, y, 10, DARKGRAY);
+        y += 30;
+
+        DrawFPS(px, GetScreenHeight() - 30);
+
+        // Update frameCount if playing
+        if (playing)
+            frameCountLocal++;
+
+        // Angle in degrees (like the p5 example)
+        int angleDeg = frameCountLocal % 360;
+
+        // Draw angle label
+        Color labelCol = WHITE;
+        DrawText(TextFormat("angle: %d", angleDeg), 25, 25, 20, labelCol);
+
+        // --- Draw circle and diameters ---
+        // circle outer
+        DrawCircleLines((int)circleX, (int)circleY, circleRadius, Fade(GRAY, 0.6f));
+        // diameters
+        DrawLine((int)circleX, (int)(circleY - circleRadius), (int)circleX, (int)(circleY + circleRadius), Fade(GRAY, 0.6f));
+        DrawLine((int)(circleX - circleRadius), (int)circleY, (int)(circleX + circleRadius), (int)circleY, Fade(GRAY, 0.6f));
+
+        // Compute moving point on circle (cos for x, sin for y). Note: cos/sin in C expect radians
+        float angRad = angleDeg * (PI / 180.0f);
```

Comment:
Please, review raylib code conventions: https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md

`*` and `/` should not have spaces